src/prototype_room_utils.js

```diff
@@ -34,7 +34,7 @@ Room.prototype.findPropertyFiltre = function(findTarget, property, properties, w
   let table = {};
```

Comment:
If you touch this, could you also quickly fix the method name to `findPropertyFilter`